model/armador_de_plantilla.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'armador_de_etiqueta_date'
+
+class ArmadorDePlantilla
+
+	def armar(plantilla) 
+		expresion = /[<]+([a-z]|[a-z]+[_])+[>]/
```

Comment:
Que este sea el primer elemento del chain of responsibility no es correcto, ahora funciona porque ninguna de las etiquetas especiales matchean con la expresión regular, pero si mañana agregamos por ejemplo la etiqueta especial: `<foo>`, este objeto va a reemplazarlo por `<%=@item.foo%>` cuando en realidad debería procesarlo el `ArmadorDeEtiquetaFoo`. Esto pasa porque están usando el manejador más genérico de la cadena en primer lugar en vez de en último.